app/controllers/mosques_controller.rb

```diff
@@ -13,6 +13,7 @@ def new
   def create
     @mosque = Mosque.new(mosque_params)
     @mosque.user_id = current_user.id # logged-in user
+    @mosque.status = "approved"
```

Comment:
```suggestion
    @mosque.status = Mosque::APPROVED
```
and make sure we have a constant for these statuses.